products/ward round documentation/new feature description_ward round documentation.md

```diff
@@ -0,0 +1,248 @@
+- Technology limitations: Manually searching for relevant data points can be hindered by slow, unreliable, or difficult to use systems.
+- Training and standardization: Variability in the training and practices of team members can lead to inconsistencies in patient care and documentation.
+- Interruptions and distractions: In case of interruptions, it is not easily comprehensible what tasks still need to be fulfilled
+
+**Purpose of ward round documentation software:**
+- Efficient record-keeping and time savings: Manual note-taking or repetitive documentation is not required and automating repetitive tasks reduces the time needed for documentation.
+- Improved communication: All team-members have access to up-to-date and consistent patient data.
+- Enhanced patient safety: Errors can be minimized by providing accurate relevant information, alerting clinicians to potentials issues such as allergies, contraindications, or pending test results.
+- Compliance: It is ensured that documentation meets regulatory and legal requirement, while standards and protocols can be upheld.
+- Integration with other systems: A holistic view of patient care and streamlined workflows are provided by integrating electronic health records, laboratory systems, imaging systems, etc.
+- Decision support: Tools that provide evidence-based recommendations, reminders, and alerts are included to assist clinicians in making informed decisions.
+- Data analysis and reporting: It enables the collection and analysis of patient data over time, facilitating the identification of trends, outcomes, and areas for improvement in patient care and hospital operations (quality management).
+- Patient engagement: Allowing for patient access or input can lead to improved quality of care and patient satisfaction.
+
+___
+
+## 3. Proposed Solution
+
+    - Describe the new feature and how it will address the problem statement.
+    - Explain the expected benefits of the feature for both medical staff and patients (e.g., improved workflow efficiency, reduced errors, enhanced patient care).
+
+**Features of “Ward Round Documentation”**
+- Interface design
+    - Modular design with automated optimized arrangement
+    - Possibility to drag and drop boxes with certain information or tools
+    - A fixed box for the ward round documentation itself should be located on the left (?) side of the screen
+- Pre-designed structure of the ward round documentation
+    - e.g. SOAP:
+        - Subjective: Current complaints reported by the patient.
+        - Objective: Current findings (physical examination, laboratory results, imaging, consultations etc.).
+        - Assessment: Brief assessment of the current problems.
+        - Procedure: Numerical list of current decisions regarding the diagnosis and treatment.
+    - Possibility to define ward-/hospital-specific templates (e.g. FAST HUGS for ICU) with semi-automated step-by-step assessment with the relevant information provided at each step
```

Comment:
maybe keep in mind that there might be different levels of documentation. I don't think we will try to keep users from defining templates, yet there will be "organizational templates" e.g. drafted by attendings that residents should be using. So we need an ownership/accountability/restriction system here.